roles/node-images-hypervisor/vars/packages/suse.yml

```diff
@@ -27,7 +27,7 @@ packages:
   - cloud-init-config-suse<23.2
   - cloud-init-doc<23.2
   - cloud-init<23.2
-  - frr=8.4-lp154.49.1
+  - frr=7.4-150300.4.12.1
```

Comment:
No, we want 8.4.

There was a new repo added, you may have to `docker rmi csm-rpms-cache-x86_64` and re-run the update script against this packages file. If you did that, then we have a zypper priority problem.